src/test/java/com/google/api/codegen/testdata/java_grpc_stub_library.baseline

```diff
@@ -0,0 +1,548 @@
+  private final UnaryCallable<GetBookRequest, Book> getBookCallable;
+  private final UnaryCallable<ListBooksRequest, ListBooksResponse> listBooksCallable;
+  private final UnaryCallable<ListBooksRequest, ListBooksPagedResponse> listBooksPagedCallable;
+  private final UnaryCallable<DeleteBookRequest, Empty> deleteBookCallable;
+  private final UnaryCallable<UpdateBookRequest, Book> updateBookCallable;
+  private final UnaryCallable<MoveBookRequest, Book> moveBookCallable;
+  private final UnaryCallable<ListStringsRequest, ListStringsResponse> listStringsCallable;
+  private final UnaryCallable<ListStringsRequest, ListStringsPagedResponse> listStringsPagedCallable;
+  private final UnaryCallable<AddCommentsRequest, Empty> addCommentsCallable;
+  private final UnaryCallable<GetBookFromArchiveRequest, BookFromArchive> getBookFromArchiveCallable;
+  private final UnaryCallable<GetBookFromAnywhereRequest, BookFromAnywhere> getBookFromAnywhereCallable;
+  private final UnaryCallable<UpdateBookIndexRequest, Empty> updateBookIndexCallable;
+  private final StreamingCallable<StreamShelvesRequest, StreamShelvesResponse> streamShelvesCallable;
+  private final StreamingCallable<StreamBooksRequest, Book> streamBooksCallable;
+  private final StreamingCallable<DiscussBookRequest, Comment> discussBookCallable;
+  private final StreamingCallable<DiscussBookRequest, Comment> monologAboutBookCallable;
+  private final UnaryCallable<FindRelatedBooksRequest, FindRelatedBooksResponse> findRelatedBooksCallable;
+  private final UnaryCallable<FindRelatedBooksRequest, FindRelatedBooksPagedResponse> findRelatedBooksPagedCallable;
+  private final UnaryCallable<AddTagRequest, AddTagResponse> addTagCallable;
+  private final UnaryCallable<AddLabelRequest, AddLabelResponse> addLabelCallable;
+  private final UnaryCallable<GetBookRequest, Operation> getBigBookCallable;
+  private final OperationCallable<GetBookRequest, Book, GetBigBookMetadata, Operation> getBigBookOperationCallable;
+  private final UnaryCallable<GetBookRequest, Operation> getBigNothingCallable;
+  private final OperationCallable<GetBookRequest, Empty, GetBigBookMetadata, Operation> getBigNothingOperationCallable;
+  private final UnaryCallable<TestOptionalRequiredFlatteningParamsRequest, TestOptionalRequiredFlatteningParamsResponse> testOptionalRequiredFlatteningParamsCallable;
+  public static final GrpcLibraryServiceStub create(LibrarySettings settings) throws IOException {
+    return new GrpcLibraryServiceStub(settings, ClientContext.create(settings));
+  }
+
+  public static final GrpcLibraryServiceStub create(ClientContext clientContext) throws IOException {
+    return new GrpcLibraryServiceStub(LibrarySettings.defaultBuilder().build(), clientContext);
+  }
+
+  /**
+   * Constructs an instance of GrpcLibraryServiceStub, using the given settings.
+   * This is protected so that it easy to make a subclass, but otherwise, the static
```

Comment:
Hmm, this comment migrated from the client template... Maybe people should subclass the stub interface instead, but I'm perfectly fine with them subclassing the Grpc stub for mocking.